frontend/src/lib/lemon-ui/LemonFileInput/LemonFileInput.tsx

```diff
@@ -43,11 +43,12 @@ export function useUploadFiles({
             try {
                 setUploading(true)
                 const formData = new FormData()
-                let blob: Blob = filesToUpload[0]
-                if (blob.type.startsWith('image/')) {
-                    blob = await lazyImageBlobReducer(blob)
+                let file: File = filesToUpload[0]
+                if (file.type.startsWith('image/')) {
+                    const compressedBlob = await lazyImageBlobReducer(file)
+                    file = new File([compressedBlob], file.name, { type: compressedBlob.type })
```

Comment:
The only reason not to do this (and it might not be) is you avoid a set of potential vulnerabilities by not accepting the user provided file name. And I can't remember what we do on the backend to mitigate that 😅

Reply:
it looks like we "only" store the file name in the DB as a string so I guess we're ok...

Reply:
Ah, wait, I see - we used to accept the file name but now it's always blob 🤦